integration_tests/testsuite/test_runtime_image.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import subprocess
+import sys
+
+import deploy_app
+import test_util
+
+
+def _deploy_app(appdir):
```

Comment:
The `deploy_app._deploy_app()` method does essentially the same thing as this, so let's try not to duplicate code as best we can. See if you can modify the existing method to be a helper that you can call from here. It's ok to change around the structure just a bit (for example, the existing one takes a base image and subs that into the existing Dockerfile: you can pull that into a helper method, then skip that part when you call it from this code).